src/eva/vision/data/datasets/classification/camelyon16.py

```diff
@@ -161,7 +161,7 @@ def annotations(self) -> Dict[str, str]:
 
     @override
     def prepare_data(self) -> None:
-        _validators.check_dataset_exists(self._root, True)
+        _validators.check_dataset_exists(self._root, False)
```

Comment:
maybe we should change the message? now if the dataset does not exists, what is the message that we are getting?

Reply:
We're getting this message, because the `download_available` arg was set to `True`. 

```
https://github.com/kaiko-ai/eva/blob/b51c9ba00ce6ae44912f3db09492d3f69585b38d/src/eva/vision/data/datasets/_validators.py#L58
```

Reply:
ah you are right! actually i wasnt aware about this addition 🙏